docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding.md

```diff
@@ -0,0 +1,146 @@
+
+### 2) Operator: Verify (the KYC process) and register a merchant
+
+1. Gather data a merchant provides, such as a registration number and the company behind the merchant, including representatives' information and their IDs.
+2. To identify the trust level and the need for additional information, validate your merchant through public registers. 
+
+{% info_block infoBox %}
+
+To accelerate the validation process, you can use relevant Spryker technology partners. For more details, reach out to your Spryker contact.
+
+{% endinfo_block %}
+
+The official relationship between the operator and merchant can be established by providing a contract to a merchant. For specific merchants, custom contracts can be used. Otherwise, boilerplate agreements must be available. Contracts can be made as part of terms and conditions that must be accepted prior, together with platform rules, customer communication guidelines, data policy, and SLA agreement. Clarification is also required regarding who the product data owner is, in which categories the merchant is allowed to sell, and the commissions and fees the merchant must pay the operator.
+
+![edit-merchant](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-2-edit-merchant.png)
+
+### 3) Operator: Approve the merchant
+
+1. In the Back Office, navigate to **Marketplace&nbsp;<span aria-label="and then">></span> Merchants**.
+2. On the **Overview of Merchants** page that opens, in the **Actions** column, click **Approve Access**. 
+
+By approving the merchant, the operator enables merchant users to access the Merchant Portal to complete their public-facing profile and create products and offers. 
+A merchant admin user needs to be created by the operator based on data provided through the registration form, such as the email or first and last name of the contact person. 
+The email with the password is automatically sent to access the Merchant Portal later on. The operator can also create more user accounts for the merchant if required.
+
+![overview-of-merchants](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-3-overview-of-merchants.png)
+
+![edit-user](https://spryker.s3.eu-central-1.amazonaws.com/docs/marketplace/user/intro-to-spryker-marketplace/merchant-onboarding/step-3-edit-user.png)
+
+### 4) Merchant and operator: Create and update the product and offers
+
+Operator: Decide who the owner of the product data is:
+* If the operator is the owner and manages the product data, other merchants can create offers with specific prices and available quantities on top of existing products.
```

Comment:
what do you mean by "on top of existing products"?

Reply:
@mikost7 question for you

Reply:
that means that the offers are created on top of the products where there is no need to provide product-specific information like name, description or image…